honeybee_radiance_command/rpict.py

```diff
@@ -0,0 +1,146 @@
+"""rpict command."""
+
+from honeybee_radiance.view import View
+from .options.rpict import RpictOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Rpict(Command):
+    """rpict command."""
+
+    __slots__ = ('output_name', 'octree', 'view', 'sky_file', 'simulation_type')
+
+    def __init__(self, options=RpictOptions(), output=None, octree=None,
+                 output_name='unnamed.hdr', view=None, sky_file=None, simulation_type=0):
```

Comment:
1. `sky_file1 should be part of the octree.
2. The default should be set up under the property not here.
3. We don't have an input for output_name. That's what output is for.
4. `simulaion_type` is something that one can expose in other libraries. Inside the command there is no such an input. We can have `classmethods` under options to generate options for a specific type of simulation. Here is an example: https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/rtrace.py#L133-L152

Reply:
I don't think I should have a class method to just modify one rpictoption. It would be better to do this downstream in other libraries.

Reply:
Either should work. This way all the libraries can use the same options.